src/main/java/models/Match.java

```diff
@@ -0,0 +1,22 @@
+package models;
+
+import java.util.Random;
+
+public class Match {
+
+    private Team team1;
+    private Team team2;
+
+    public Match(Team team1, Team team2){
+        this.team1 = team1;
+        this.team2 = team2;
+    }
+
+    public Team getMatchWinner(){

```

Comment:
A comment explaining what happens if there is a tie or just in general how the match winner is picked might be useful.